client/tests/common/mod.rs

```diff
@@ -0,0 +1,62 @@
+// Copyright 2023 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+#![allow(clippy::redundant_pub_crate, dead_code)]
+
+mod constants;
+
+use iota_client::{
+    constants::SHIMMER_COIN_TYPE, node_api::indexer::query_parameters::QueryParameter, request_funds_from_faucet,
+    secret::SecretManager, Client, Result,
+};
+
+pub use self::constants::{FAUCET_URL, NODE_LOCAL};
+
+/// Sets up a Client with node health ignored.
+pub fn setup_client_with_node_health_ignored() -> Client {
+    Client::builder()
+        .with_node(NODE_LOCAL)
+        .unwrap()
+        .with_ignore_node_health()
```

Comment:
do we really need ignore_nodehealth? its not in the wallet tests

Reply:
No, would just not return an error even if the node is not healthy